src/allocator.h

```diff
@@ -45,6 +46,9 @@ namespace ncnn {
 #define MALLOC_ALIGN 16
 #endif
 
+// For marking allocated things as aligned. Will evaluate twice on GCC!
+#define NCNN_ASSUME_ALIGNED(ptr) HEDLEY_ASSUME_ALIGNED(ptr, MALLOC_ALIGN)
+
```

Comment:
unused macro

Reply:
I tried to use it, but see my comments in Mat.h. I could not really proved that the derived rows or columns are aligned. I would really with to be able to use it.